host_os.py

```diff
@@ -56,14 +55,15 @@
               "script: %s" % ", ".join(missing_packages))
         sys.exit(MISSING_PACKAGES_ERROR)
 
-    if os.getuid() is 0 and subcommand != 'set-env':
-        print("Please, do not run this command as root, run "
-              "host_os.py set-env --user <YOUR_USER_LOGIN> command in order to "
-              "properly setup user and directory for build scripts")
+    if os.getuid() is 0:
+        print("Please, do not run this command as root.")
         sys.exit(TOO_MUCH_PRIVILEGE_ERROR)
 
-    if os.getuid() is not 0 and subcommand == 'set-env':
-        print("The set-env command should be run with root privileges")
+    mock_users = grp.getgrnam('mock').gr_mem
+    user = os.environ['USER']
+    if user not in mock_users:
+        print("User must be in mock group, please run "
+              "'sudo usermod -a -G mock $(whoami)'")
```

Comment:
Having the user in the 'mock' group is required for running subcommands which do not use mock (eg elease-notes, upgrade-versions and update-versions-readme) ? If not, why block the user from executing them?